ridbay/UI
# #165146273 Update landing page

#### What does this PR do?
This PR updates the landing page of the UI.

#### Description of Task to be completed?
The user should visit the landing page URL, where should see links to internal pages.

#### How should this be manually tested?
The user should visit the landing page URL, User should click on each page links to visit the internal pages.
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[User can access all pages from the landing page](https://www.pivotaltracker.com/story/show/165146273)

#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
